Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/18.4/InspectorBackendCommands.js

```diff
@@ -0,0 +1,589 @@
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+ * THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+ * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+// DO NOT EDIT THIS FILE. It is automatically generated from CombinedDomains.json by the script: Source/JavaScriptCore/inspector/scripts/generate-inspector-protocol-bindings.py
+
+// Animation
+InspectorBackend.registerDomain("Animation", ["page"]);
+InspectorBackend.registerEnum("Animation.AnimationState", {Ready: "ready", Delayed: "delayed", Active: "active", Canceled: "canceled", Done: "done"});
+InspectorBackend.registerEnum("Animation.PlaybackDirection", {Normal: "normal", Reverse: "reverse", Alternate: "alternate", AlternateReverse: "alternate-reverse"});
+InspectorBackend.registerEnum("Animation.FillMode", {None: "none", Forwards: "forwards", Backwards: "backwards", Both: "both", Auto: "auto"});
+InspectorBackend.registerCommand("Animation.enable", null, [], []);
+InspectorBackend.registerCommand("Animation.disable", null, [], []);
+InspectorBackend.registerCommand("Animation.requestEffectTarget", null, [{"name": "animationId", "type": "string"}], ["effectTarget"]);
+InspectorBackend.registerCommand("Animation.resolveAnimation", null, [{"name": "animationId", "type": "string"}, {"name": "objectGroup", "type": "string", "optional": true}], ["object"]);
+InspectorBackend.registerCommand("Animation.startTracking", null, [], []);
+InspectorBackend.registerCommand("Animation.stopTracking", null, [], []);
+InspectorBackend.registerEvent("Animation.animationCreated", null, ["animation"]);
+InspectorBackend.registerEvent("Animation.nameChanged", null, ["animationId", "name"]);
+InspectorBackend.registerEvent("Animation.effectChanged", null, ["animationId", "effect"]);
+InspectorBackend.registerEvent("Animation.targetChanged", null, ["animationId"]);
+InspectorBackend.registerEvent("Animation.animationDestroyed", null, ["animationId"]);
+InspectorBackend.registerEvent("Animation.trackingStart", null, ["timestamp"]);
+InspectorBackend.registerEvent("Animation.trackingUpdate", null, ["timestamp", "event"]);
+InspectorBackend.registerEvent("Animation.trackingComplete", null, ["timestamp"]);
+InspectorBackend.registerAnimationDispatcher = InspectorBackend.registerDispatcher.bind(InspectorBackend, "Animation");
+InspectorBackend.activateDomain("Animation", ["page", "web-page"]);
+
+// ApplicationCache
```

Comment:
all mentions of `ApplicationCache` and `appcache` and etc. everywhere in this PR should be removed since the frontend no longer supports that as of [289820@main](https://commits.webkit.org/289820@main)